demo/TwoLists/src/buildInteractionModel.ts

```diff
@@ -109,6 +108,7 @@ export namespace TwoListsIM {
             ],
         })
 
+        .addOrMergeSlotTypes({ name: 'target', values: [] })
```

Comment:
This isn't required, 
`.addValuesToSlotType()` below add slot types too, did you get any errors without this line ??